### ISSUE TITLE
Allow to limit the number of tabs for auto-lock entries

### ISSUE THREAD

**Issue by hariseldon78** [tabwrangler/tabwrangler#221]
I think could be a useful feature to allow for each auto-lock entry a maximum number of tabs. The app could then close the older tabs when that number is reached.

**Comment by ssorallen** [comment]
Interesting idea, that sounds cool. The "Auto-Lock" UI would need some improvements in order to make that apparent. I'd be interested to see how we could make this work.

**Comment by hariseldon78** [comment]
I could find some time in the next few weeks to prepare a pull request, does somebody have any directions on where to put my hands on or where to start studying the code?

**Comment by ssorallen** [comment]
@hariseldon78 Any contribution is appreciated. I'm happy to help and give you feedback once you start looking at this. I suggest looking at where the table is rendered in the OptionsTab since that also tells you where the setting comes from:

https://github.com/tabwrangler/tabwrangler/blob/d5e784499bc0e223db8aebe048998ed6a603eb58/app/js/OptionsTab.js#L417-L448

You could maybe add a new column to the table and a new form control for defining the max number of tabs. I would consider using a dropdown with a small set of possibilities where the default is "No limit". You could do the first few numbers of the Fibonacci sequence: 1, 2, 3, 5, 8, 13.

**Comment by KTamas** [comment]
yeah, it's interesting but for now, i'll put it on the backburner. PRs welcome, per usual.
